pandas/core/sparse/frame.py

```diff
@@ -74,6 +75,8 @@ def __init__(
         dtype=None,
         copy=False,
     ):
+        if not is_scalar(default_fill_value):
```

Comment:
Is a dict acceptable here at all? Think we allow that for normal DataFrame fillna

Reply:
@TomAugspurger are there cases where non-scalar is accepted?  I was getting errors in another branch caused by zero-dim ndarray getting through here.

Reply:
I don't believe we allow this (nor should)